.gitlab/os.yml

```diff
@@ -12,19 +12,22 @@
     ARCH: 'toss_3_x86_64_ib'
     GCC_VERSION: '8.3.1'
     CLANG_VERSION: '9.0.0'
+    SPHERAL_BUILDS_DIR: /p/lustre1/sphapp/spheral-ci-builds
   extends: [.sys_config]
```

Comment:
Is this architecture still used anywhere?

Reply:
I don't think so.

Reply:
Is it worth removing?

Reply:
We might have other references to toss3. I'll add it as an issue to remember for the sake of getting this merged into everyone's branches ASAP.